runelite-client/src/main/java/net/runelite/client/plugins/chatnotifications/ChatNotificationsPlugin.java

```diff
@@ -124,6 +126,30 @@ private void updateHighlights()
 		}
 	}
 
+	/**
+	 * Remove tags, except for &lt;lt&gt; and &lt;gt&gt;
+	 *
+	 * @return
+	 */
+	private static String removeTags(String str)
```

Comment:
Instead of copying this from EmojiPlugin, move the method from there to `Text` (naming it `removeFormattingTags`) and make it public in a commit prior to the current changes. (and add tests for it--see `TextTest`'s removeTags test cases)